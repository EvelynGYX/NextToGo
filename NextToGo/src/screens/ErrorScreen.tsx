import React from 'react';
import {StyleSheet, Text, View} from 'react-native';

const ErrorScreen = () => {
  return (
    <View style={styles.container}>
      <Text>Error</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
  },
});

export default ErrorScreen;
