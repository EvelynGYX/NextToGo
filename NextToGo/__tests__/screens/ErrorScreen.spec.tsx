import React from 'react';
import renderer from 'react-test-renderer';
import ErrorScreen from '../../src/screens/ErrorScreen';

describe('ErrorScreen', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ErrorScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
