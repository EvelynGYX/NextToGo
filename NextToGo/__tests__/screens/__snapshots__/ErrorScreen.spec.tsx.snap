// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorScreen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Error
  </Text>
</View>
`;
